pyproject.toml

```diff
@@ -22,6 +22,7 @@ pandas = "^2.1.0"
 StrEnum = "^0.4.15"
 # pyarrow 14.0.2 doesn't limit numpy < 2, but it should, we do it here
 numpy = "<2"
+types-requests = "^2.32.0.20241016"
```

Comment:
Where is this used?

Reply:
It's used by mypy; maybe it could just be a dev dependency. We have it as a normal dependency in `studio-backend`.